tests/execution_engine/conftest.py

```diff
@@ -210,3 +210,19 @@ def generate_ascending_list_of_datetimes(
         "timestamp", F.col("timestamp").cast(IntegerType()).cast(StringType())
     )
     return spark_df
+
+
+@pytest.fixture
+def test_folder_connection_path_tsv(tmp_path_factory):
```

Comment:
`tmp_path_factory` is session-scoped. Not sure if it causes an issue here but can we use `tmp_path`?

Reply:
Additionally, could we type annotate this (particularly the return)? Type annotations are strictly necessary in our tests but are a nice-to-have 😄

Reply:
@cdkini How about in a separate PR?  The scope of this one was to move test methods.  Thank you!